linode/manager
# upcoming: [M3-8420] – Add conditionally-displayed reboot notice on Volume Create page

## Description 📝
Add conditionally-displayed reboot notice on Volume Create page

## Target release date 🗓️
9/16/24

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![Screenshot 2024-09-10 at 11 29 06 AM](https://github.com/user-attachments/assets/ff2cd5ac-6652-48f5-bd21-7d0f23a1e8e6) | ![Screenshot 2024-09-10 at 11 28 14 AM](https://github.com/user-attachments/assets/1bf13b37-de1c-44e1-aac1-55f076f041c5) |


## How to test 🧪
### Prerequisites
Point at the dev environment with the `blockstorage-encryption` tag on your account.

### Verification steps
Pointing at the dev environment:

With the BSE feature flag off, you should not see a client library update reboot notice regardless of which linode you select.

With the BSE feature flag on, if you select a linode that does not have the `blockstorage_encryption` capability & check the "Encrypt Volume" checkbox, you should see the client library update reboot notice below the Linode dropdown. The "Create Volume" button should also be disabled.

Confirm the alignment of the Linode & Config dropdowns look good in both cases, and across viewport widths (they stack in mobile view)

## As an Author I have considered 🤔
